refs/biblio.json

```diff
@@ -780,36 +780,54 @@
         "etAl": true,
         "href": "http://dublincore.org/documents/dc-rdf/",
         "title": "Expressing Dublin Core metadata using the Resource Description Framework (RDF)",
-        "date": "14 January 2008",
+        "rawdate": "2008-01-14",
```

Comment:
It's `rawDate`, not `rawdate`.

Reply:
Sorry for the syntax error, @tobie . Just fixed it.